app/src/main/java/com/github/se/cyrcle/model/review/Review.kt

```diff
@@ -0,0 +1,9 @@
+package com.github.se.cyrcle.model.review
+
+data class Review(
+    val uid: String,
+    val owner: String,
+    val text: String,
+    val rating: Double,
```

Comment:
I think this will make more sense when I show the screen too. The idea was for the screen to display something similar to stars, and I've personally seen a good amount of apps allow half-stars. I'll leave it as a double for now and we can decide whether or not we make that change then